### PR TITLE
add stft_frame_index_to_sample_index

### PR DESCRIPTION
**Pull request by JanekEbb**

### CODE REVIEW

File: paderbox/transform/module_stft.py [hunk 1]
```diff
@@ -378,6 +378,73 @@ def sample_index_to_stft_frame_index(sample, window_length, shift, fading='full'
...
+    """Computes first, center or last sample index from frame index
+
+    Args:
+        frame_index:
+        window_length: stft window length
+        shift: stft hop size
+        pad: True if stft uses padding else False
+        fading: fading used in stft
+        mode: states the sample to return \in {'first','center','last'}. Default is 'center'.
+        num_samples: total number of samples in the source signal
```
Comment (jensheit) [inline paderbox/transform/module_stft.py:393]: Perhaps add the information that num_samples allows for a negativ frame_index

**Review by jensheit — APPROVED** [review]

File: paderbox/transform/module_stft.py [hunk 2]
```diff
@@ -378,6 +378,73 @@ def sample_index_to_stft_frame_index(sample, window_length, shift, fading='full'
     return frame
 
 
+def stft_frame_index_to_sample_index(
+        frame_index, window_length, shift, pad=True, fading=None, mode='center', num_samples=None
```
Comment (boeddeker) [inline paderbox/transform/module_stft.py:382]: Can you match the default for `fading` with the `stft`?

File: paderbox/transform/module_stft.py [hunk 3]
```diff
@@ -378,6 +378,73 @@ def sample_index_to_stft_frame_index(sample, window_length, shift, fading='full'
...
+):
+    """Computes first, center or last sample index from frame index
+
+    Args:
+        frame_index:
+        window_length: stft window length
+        shift: stft hop size
+        pad: True if stft uses padding else False
+        fading: fading used in stft
+        mode: states the sample to return \in {'first','center','last'}. Default is 'center'.
```
Comment (boeddeker) [inline paderbox/transform/module_stft.py:392]: Can you mention, that you round up for `center`?

**Review by thequilo — APPROVED** [review]
I think that this function doesn't work with PyTorch tensors on the GPU, but I think that that's fine.

**Review by TCord — APPROVED** [review]

**Review by boeddeker — APPROVED** [review]

**Pull request merged**